src/storybook/stand-alone-documentaion/welcome/contributor/contributor.jsx

```diff
@@ -12,9 +12,9 @@ export const Contributor = ({ imgSrc, className, fullName, title, email }) => {
     <Flex justify={Flex.justify.SPACE_BETWEEN}>
       <InformationBoxTitle>{fullName}</InformationBoxTitle>
       {email && (
-        <a tabIndex="-1" href={`mailto:${email}`}>
+        <Link tabIndex="-1" href={`mailto:${email}`} target={Link.targets.SELF}>
```

Comment:
I think you need to update storybook-components' version because targets is null